lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.py

```diff
@@ -34,22 +35,26 @@ def defaultVariantSetName():
 def defaultVariantName():
     return kDefaultVariantName
 
-def createPrimAndAttributes(stage, primPath, mayaReferencePath, mayaNamespace):
+def createPrimAndAttributes(stage, primPath, mayaReferencePath, mayaNamespace, mayaAutoEdit):
     prim = stage.DefinePrim(primPath.path, 'MayaReference')
 
     mayaReferenceAttr = prim.CreateAttribute('mayaReference', Sdf.ValueTypeNames.Asset)
     mayaReferenceAttr.Set(mayaReferencePath)
 
     mayaNamespaceAttr = prim.CreateAttribute('mayaNamespace', Sdf.ValueTypeNames.String)
     mayaNamespaceAttr.Set(mayaNamespace)
+    
+    mayaAutoEditAttr = prim.CreateAttribute('mayaAutoEdit', Sdf.ValueTypeNames.Bool)
```

Comment:
I saw below that there are tokens for these strings (mayaAutoEdit, mayaNamespace & mayaReference). It would be great it we used the tokens here in instead of hard-coding.

Reply:
There is a bigger cleanup to do around these. See other places, e.g. https://github.com/Autodesk/maya-usd/blob/70124fe61c82bbafc49393101c012a85c7b47ecc/lib/mayaUsd/fileio/translators/translatorMayaReference.cpp#L141 that requires cleanup. Not in scope for this PR.